docs/user/openstack/README.md

```diff
@@ -93,7 +93,7 @@ openstack object store account set --property Temp-URL-Key=superkey
 
 ### Disk Requirements
 
-Etcd is run on your control plane nodes, and it has disk requirements that need to be met to ensure the stability of your cluster. It is important that you make sure that the ephemeral disk that gets attached to instances of your chosen flavor meets [these requirements](https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware.md#disks). If this is not the case, then please refer to the known issue documentation [here](known-issues.md#boot-from-volume-support).
+Etcd runs on the control plane nodes, and has disk requirements that need to be met to ensure the stability of the cluster. If the ephemeral disk that gets attached to instances of the chosen flavor does not meet [etcd requirements](https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware.md#disks), check if the cloud has a more performant volume type and use a [custom `install-config.yaml`](customization.md) to deploy the cluster with root volumes.
```

Comment:
Nit: "the cluster with root volumes" -> "the control plane with root volumes".  Compute nodes don't come into this.